README.md

```diff
@@ -60,73 +44,37 @@ cd sdk-workspace
-git clone -b <CRT_VERSION> https://github.com/awslabs/aws-crt-python.git
-
-# 3. Update the submodules
-cd aws-crt-python
-git submodule update --init --recursive
-
-# 4. (Optional) Setup the version number of your local build. Similar to the awsiotsdk, the default version for awscrt is set to "1.0.0-dev", you can set the version number of the local build in "./aws-crt-python/awscrt/__init__.py". The awscrt version set here need to match the version specified in "./aws-iot-device-sdk-python-v2/setup.py" so that the awsiotsdk could locate the correct awscrt library.
-sed -i "s/__version__ = '1.0.0.dev0'/__version__ = '<CRT_VERSION>'/" awscrt/__init__.py
-
-# 5. Install using Pip
-python3 -m pip install .
-```
-If you need aws-crt-python to use the libcrypto included on your system, set environment variable AWS_CRT_BUILD_USE_SYSTEM_LIBCRYPTO=1 while building from source:
-```
-AWS_CRT_BUILD_USE_SYSTEM_LIBCRYPTO=1 python3 -m pip install --no-binary :all: --verbose awscrt
-```
-( --no-binary :all: ensures you do not use the precompiled wheel from PyPI)
-
-If you encounter issues, see [Installation Issues](./documents/PREREQUISITES.md#installation-issues) and try again.
-
 ## Samples
 
 [Samples README](samples)
 
 ## Getting Help
 
-The best way to interact with our team is through GitHub. You can [open an issue](https://github.com/aws/aws-iot-device-sdk-python-v2/issues) and choose from one of our templates for guidance, bug reports, or feature requests. You may also find help on community resources such as [StackOverFlow](https://stackoverflow.com/questions/tagged/aws-iot) with the tag #aws-iot or If you have a support plan with [AWS Support](https://aws.amazon.com/premiumsupport/), you can also create a new support case.
+The best way to interact with our team is through GitHub. You can open a [discussion](https://github.com/aws/aws-iot-device-sdk-python-v2/discussions) for guidance questions or an [issue](https://github.com/aws/aws-iot-device-sdk-python-v2/issues/new/choose) for bug reports, or feature requests. You may also find help on community resources such as [StackOverFlow](https://stackoverflow.com/questions/tagged/aws-iot) with the tag [#aws-iot](https://stackoverflow.com/questions/tagged/aws-iot) or if you have a support plan with [AWS Support](https://aws.amazon.com/premiumsupport/), you can also create a new support case.
 
 Please make sure to check out our resources too before opening an issue:
 
-* Our [FAQ](./documents/FAQ.md)
-* [API documentation](https://aws.github.io/aws-iot-device-sdk-python-v2/)
-* Our [Developer Guide](https://docs.aws.amazon.com/iot/latest/developerguide/what-is-aws-iot.html) ([source](https://github.com/awsdocs/aws-iot-docs))
+* [FAQ](./documents/FAQ.md)
+* [API Docs](https://aws.github.io/aws-iot-device-sdk-python-v2/)
+* [Developer Guide](https://docs.aws.amazon.com/iot/latest/developerguide/what-is-aws-iot.html) ([source](https://github.com/awsdocs/aws-iot-docs))
```

Comment:
Could we use `IoT Instruction` or `IoT Guide` instead of `Developer Guide`? `Developer Guide` sounds like the SDK Developer Guide, while the link actually points to the IoT developer guide and does not mention much about the SDK details.